lighthouse-core/config/default.json

```diff
@@ -59,8 +58,7 @@
     "image-alt",
     "label",
     "tabindex",
-    "content-width",
-    "geolocation-on-start"
```

Comment:
so the audit name reference here is "geolocation-on-start" and the `meta.name` of the actual audit was just "geolocation", which i suppose was our problem.

do you want to find out why this validation check wasn't throwing?
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/config.js#L189-L191